2022-06-14 14:05:38.254 [INFO ] io.netty.util.internal.logging.AbstractInternalLogger.log[217 line]  [SerialNo: ] [message]   - null Unexpected exception during request: java.net.SocketException: Connection reset
java.net.SocketException: Connection reset
	at sun.nio.ch.SocketChannelImpl.throwConnectionReset(SocketChannelImpl.java:394) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:426) ~[?:?]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:253) ~[netty-buffer-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:350) ~[netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) [netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719) [netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) [netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) [netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-transport-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) [netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.70.Final.jar:4.1.70.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.70.Final.jar:4.1.70.Final]
	at java.lang.Thread.run(Thread.java:833) [?:?]
2022-06-14 14:05:38.374 [INFO ] io.netty.util.internal.logging.AbstractInternalLogger.log[171 line]  [SerialNo: ] [message]   - Reconnecting, last destination was /127.0.0.1:6379
2022-06-14 14:05:38.389 [WARN ] io.netty.util.internal.logging.AbstractInternalLogger.log[151 line]  [SerialNo: ] [message]   - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: no further information: /127.0.0.1:6379
2022-06-14 14:05:43.199 [INFO ] com.alibaba.druid.pool.DruidDataSource.close[2071 line]  [SerialNo: ] [message]   - {dataSource-1} closing ...
2022-06-14 14:05:43.199 [INFO ] com.alibaba.druid.pool.DruidDataSource.close[2144 line]  [SerialNo: ] [message]   - {dataSource-1} closed
